webapp/components/test-search.js

```diff
@@ -184,12 +192,19 @@ class TestSearch extends WPTFlags(PolymerElement) {
         padding: 0.5em 0;
         width: 100%;
       }
+      .help {
+        font-size: x-small;
```

Comment:
The computed size for me is 10px. I suspect this is smaller than would get past an accessibility review, if we ever do that.

Reply:
I intend to deprecate this with a search builder in the future.